README.md

```diff
@@ -3,121 +3,102 @@
 [![CircleCI](https://circleci.com/gh/yggdrasil-network/yggdrasil-go.svg?style=shield&circle-token=:circle-token
 )](https://circleci.com/gh/yggdrasil-network/yggdrasil-go)
 
-## Introduction
+## Introdução
 
-Yggdrasil is an early-stage implementation of a fully end-to-end encrypted IPv6
-network. It is lightweight, self-arranging, supported on multiple platforms and
-allows pretty much any IPv6-capable application to communicate securely with
-other Yggdrasil nodes. Yggdrasil does not require you to have IPv6 Internet
-connectivity - it also works over IPv4.
+O Yggdrasil é uma implementação inicial de uma rede IPv6 totalmente criptografada de ponta a ponta. É leve, auto-organizado, é suportado em várias plataformas e permite que praticamente qualquer aplicativo compatível com IPv6 se comunique de forma segura com outros nós do Yggdrasil. O Yggdrasil não exige que você tenha conectividade com a Internet IPv6 - ele também funciona com IPv4.
 
-Although Yggdrasil shares many similarities with
-[cjdns](https://github.com/cjdelisle/cjdns), it employs a different routing
-algorithm based on a globally-agreed spanning tree and greedy routing in a
-metric space, and aims to implement some novel local backpressure routing
-techniques. In theory, Yggdrasil should scale well on networks with
-internet-like topologies.
+Embora Yggdrasil compartilhe muitas semelhanças com [cjdns](https://github.com/cjdelisle/cjdns), emprega um algoritmo de roteamento diferente com base em uma árvore de abrangência globalmente acordada e roteamento ganancioso em um espaço métrico, e visa implementar algumas novas técnicas de roteamento de contrapressão local. Em teoria, o Yggdrasil deve escalar bem em redes com topologias semelhantes à Internet.
```

Comment:
Extrapolating the line char limit (I think it should be 80 or 120 maximum chars per line). This violation happens throughout the whole file but I just tagged here for convenience.